docs/advanced_topics/images/custom_image_model.rst

```diff
@@ -41,6 +41,18 @@ Here's an example:
     class CustomRendition(AbstractRendition):
         image = models.ForeignKey(CustomImage, on_delete=models.CASCADE, related_name='renditions')
 
+        # Override img_tag from Abstract rendition to use your new fields
+        def img_tag(self, extra_attributes={}):
```

Comment:
As per https://github.com/wagtail/wagtail/issues/4236#issuecomment-363483065, let's leave this undocumented.